content/showcase/happyhols/contents.lr

```diff
@@ -0,0 +1,13 @@
+name: happyhols
+---
+url: https://happyhols.co.za/
+---
+cover_image: happyhols-cover.jpg
```

Comment:
This PR isn't building because this shows an image name that isn't here. `happyhols-cover.jpg` doesn't exist.

Reply:
Oops, should work now